doc/whats-new.md

```diff
@@ -1,497 +1,144 @@
-could be exceeded in large calculations. We raised that limit to over 4
-billion events per task.
-
-We fixed a long standing bug in the event based risk calculation. In some
-cases (when the hazard sites were given as a region) it was associating
-incorrectly the assets to the sites and produced bogus numbers.
-
-We fixed a couple of bugs affecting exposures with multiple assets of the
-same taxonomy on the same site: that made it impossible to run `classical_risk`
-and `scenario_risk` calculations for such exposures.
-
-We fixed an annoying encoding bug in the commands `oq engine --lhc` and
-`oq engine --lrc` which affected the display of calculations with non-ASCII
-characters in the description.
-
-We fixed a bug in `event_based_risk`: it was impossible to use
-vulnerability functions with "PM" distribution, i.e. with Probability
-Mass Functions. Now they work as expected.
-
-The .npz export for scenario calculations has been fixed in the case
-of a single event, i.e. when `number_of_ground_motion_fields=1`, which
-was broken.
-
-Additional validations
-----------------------
-
-The engine is more picky than before. For instance if an user
-specifies `quantile_loss_curves` or `conditional_loss_poes` in a
-`classical_damage` calculation she will now get an error, since such
-settings make no sense in that context. Before they were silently
-ignored.
-
-If an exposure contains assets with taxonomies for which there are no
-vulnerability functions available, now the user gets a clear error
-before starting the calculation and not in the middle of it.
-
-If an user provides a complex logic tree file which is
-invalid for the scenario calculator, now she gets a clear error message.
-
-There are more checks for patological situations, like the user 
-providing no intensity measure sites, no GSIMs, no sites: now a clearer
-error message will be displayed.
-
-Experimental new features
+This release introduces a tiling mechanism for classical PSHA
+calculations. The tiling reduces the memory consumption and increases
+the performance of large calculations: the improvement is sensible
+for calculations dominated by distance computations, less so in other
+cases.
+
+There is a new `MultiPointSource` object in hazardlib, with its own
+XML serialization protocol. Serialized MultiPointSources can be up to
+an order of magnitude smaller than equivalent PointSources.
+MultiPointSource are more efficient to read, to write and to
+transfer. The computational performance is more or less the same as
+before, though.
+
+This is the first release integrating hazardlib into the engine: there
+are no more hazardlib packages. The change is transparent to the user
+and the upgrade procedure will automatically uninstall the old
+hazardlib. However, scientists and developers using the oq-hazardlib
+repository from GitHub should know that it has been deprecated.  In
+order to avoid confusion, we suggest to remove the oq-hazardlib
+repository if you have one.
```

Comment:
I would add link to https://github.com/gem/oq-engine/blob/master/doc/faq.md#openquke-hazardlib-errors `faq.md`